src/main/java/dk/cngroup/lentils/controller/CypherController.java

```diff
@@ -26,10 +26,14 @@
     private static final String REDIRECT_CYPHER_LIST = "redirect:/admin/cypher/";
     private static final String HEADING_ADD_CYPHER = "Nová šifra";
     private static final String HEADING_EDIT_CYPHER = "Upravit šifru";
+    private static final String TEMPLATE_ATTR_HEADING = "heading";
```

Comment:
V ostatnich kontrolerech, kde jsme dělali ty konstanty jsou ty template_attr konstanty oddělené mezerou od ostatních, možná bysme to měli taky sjednotit :)